protected/widgets/menu/assets/menu.css

```diff
@@ -29,53 +29,60 @@
 }
 
 .menu {
-    overflow: hidden;
-    --animation-duration: 500ms;
-    --indicator-width: 3px;
-    --indent: 20px;
-    --item-padding-vertical: 10px;
-    padding-top: 10px;
-    position: relative;
-    background-color: white;
-    grid-area: sidebar;
-    font-size: 1em;
-    z-index: 10000;
-    display: grid;
-    grid-template-columns: 1fr;
-    grid-template-rows: auto auto auto 1fr auto;
-    grid-template-areas:
-            "image"
-            "title"
-            "separator"
-            "nav"
-            "footer"
-}
-
-@keyframes fadein {
-    from { opacity: 0; }
-    to   { opacity: 1; }
+  --indicator-width: 3px;
+  --indent: 20px;
+  --item-padding-vertical: 14px;
+  overflow: hidden;
+  padding-top: 10px;
+  position: relative;
+  border-radius: 5px;
+  border-top-right-radius: 0px;
+  background-color: white;
+  grid-area: sidebar;
+  font-size: 1em;
+  z-index: 10000;
+  display: grid;
+  height: 100%;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto 1fr auto;
+  grid-template-areas: "image" "title" "separator" "nav" "footer";
```

Comment:
You should not randomly reformat code, it makes changes harder to find.
In the case of `grid-template-areas` reformatting makes it less readable; by using new lines you actually make sure that the value is an ascii art representation of your layout.

https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas